translationCoreApps/translationWords_Check_plugin
# Tests

#### This pull request addresses:

Added code coverage, linting, and stubbed out failing test.

#### How to test this pull request:

I moved files around so make sure the code still runs!
The tests should fail because there aren't any real tests yet.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationwords_check_plugin/103)
